src/formatters/proseFormatter.ts

```diff
@@ -20,6 +20,10 @@ import {RuleFailure} from "../language/rule/rule";
 
 export class Formatter extends AbstractFormatter {
     public format(failures: RuleFailure[]): string {
+        if (!failures.length) {
```

Comment:
TSLint norms is to do an explicit comparison, `(failures.length === 0)`.

Just curious, do you have that feature enabled that lets TSLint maintainers make modifications to your PR branch? I haven't gotten a chance to try it out yet, and something like this is the ideal scenario